src/Tags/DescriptionTag.php

```diff
@@ -11,13 +11,13 @@ public static function initialize(?SEOData $SEOData): ?MetaTag
     {
         $description = $SEOData?->description;
 
-        if (! $description) {
+        if (!$description) {
             return null;
         }
 
         return new MetaTag(
             name: 'description',
-            content: trim($description)
```

Comment:
trim is now done on every tag attribute by default directly in `->collectAttributes`